content/en/security/application_security/enabling/compatibility/nodejs.md

```diff
@@ -6,46 +6,163 @@ type: multi-code-lang
+When the Node.js project drops support for an LTS major release line (when it goes End of Life), support for it is dropped in the next major version of `dd-trace`.
+The last major supporting release line of `dd-trace` library supports that EOL version of Node.js for at least another year on a maintenance mode basis.
 
+Some issues cannot be solved in `dd-trace` and instead must be solved in Node.js. When this happens and the Node.js release in question is EOL, it's not possible to solve the issue without moving to another non-EOL release.
+Datadog does not make new releases of `dd-trace` to provide specific support for non-LTS Node.js major release lines (odd numbered versions).
 
-These are supported on the following architectures:
+For the best level of support, always run the latest LTS release of Node.js, and the latest major version of `dd-trace`. Whatever release line of Node.js you use, also use the latest version of Node.js on that release line, to ensure you have the latest security fixes.
 
-- Linux (GNU) x86-64
-- Alpine Linux (musl) x86-64
-- macOS (Darwin) x86-64
-- Windows (msvc) x86, x86-64
+For more information about Node.js release, see the [official Node.js documentation][4].
+
+
+
+### Operating system support
+
+The following operating systems are officially supported by `dd-trace`. Any operating system not listed is still likely to work, but with some features missing, for example ASM, profiling, and runtime metrics. Generally speaking, operating systems that are actively maintained at the time of initial release for a major version are supported.
 
 You can monitor application security for Node.js apps running in Docker, Kubernetes, AWS ECS, and AWS Fargate.
 
-### Supported frameworks
+| dd-trace Version    | Operating System      | Architectures         | Minimum Versions                         |
+| ------------------- | --------------------- | --------------------- | ---------------------------------------- |
+| 3.x                 | Linux (glibc)         | arm, arm64, x64       | CentOS 7, Debian 9, RHEL 7, Ubuntu 14.04 |
+|                     | Linux (musl)          | arm, arm64, x64       | Alpine 3.13                              |
+|                     | macOS                 | arm64, x64            | Catalina (10.15)                         |
+|                     | Windows               | ia32, x64             | Windows 8.1, Windows Server 2012         |
+| 2.x                 | Linux (glibc)         | arm, arm64, ia32, x64 | CentOS 7, Debian 9, RHEL 7, Ubuntu 14.04 |
+|                     | Linux (musl)          | arm, arm64, ia32, x64 | Alpine 3.10                              |
+|                     | macOS                 | arm64, x64            | Yosemite (10.10)                         |
+|                     | Windows               | ia32, x64             | Windows 8.1, Windows Server 2012         |
```

Comment:
we don't support arm32 (`arm`) nor ia32 in linux

Reply:
updated this section